README.md

```diff
@@ -190,7 +205,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [BetterTouchTool](http://www.boastr.net/)
 - [BibDesk](http://bibdesk.sourceforge.net/)
 - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
-- [Bitchx](http://www.bitchx.org/)
+- [Jerkx](http://www.jerkx.org/)
```

Comment:
Not sure it's related to this PR

Reply:
I suspect there were hidden characeters that my text editor automatically corrected

Reply:
Well I don't know, but it's still here.